src/routes/namespaces/[namespace]/workflows/_workflows-summary-table.svelte

```diff
@@ -1,18 +1,17 @@
-<table class="border-collapse w-full">
-  <thead>
-    <tr class="bg-gray-200">
-      <th class="w-3/12 text-left ">Workflow Name</th>
-      <th class="w-3/12 text-left">Workflow ID</th>
-      <th class="w-1/12">Status</th>
-      <th class="w-2/12 text-right">Started</th>
-      <th class="w-2/12 text-right">Ended</th>
-    </tr>
-  </thead>
-  <slot name="rows" />
-</table>
+<section class="workflow-table border-collapse w-full rounded-lg">
+  <div class="bg-gray-200 flex flex-row p-2">
+    <div class="w-3/12 text-left">Workflow ID</div>
+    <div class="w-3/12 text-left">Type</div>
+    <div class="w-3/12 text-left">Status</div>
+    <div class="w-2/12 text-left">Started</div>
+    <div class="w-2/12 text-left">End</div>
+  </div>
+  <slot />
+</section>
 
 <style lang="postcss">
-  th {
-    @apply bg-gray-200 text-gray-500 text-xs h-6 m-0 p-3 uppercase table-fixed;
+  .workflow-table {
+    display: block;
```

Comment:
I'm not saying we should _never_ write custom CSS properties that aren't from Tailwind, but it's definitely something we want to avoid when possible.

There is a Tailwind class called `block` that you can just toss into the list of classes above.

Reply:
But even then–it's a `section`, which should be a block by default.